extensions/postgres-flyway/src/main/resources/migration/default/V2__MS8-to-0.1.0.sql

```diff
@@ -0,0 +1,93 @@
+--
+--  Copyright (c) 2023 sovity GmbH
+--
+--  This program and the accompanying materials are made available under the
+--  terms of the Apache License, Version 2.0 which is available at
+--  https://www.apache.org/licenses/LICENSE-2.0
+--
+--  SPDX-License-Identifier: Apache-2.0
+--
+--  Contributors:
+--       sovity GmbH - Update Tables From MS8 to 0.1.0 EDC
+--
+--
+
+-- Assets
+alter table edc_asset_property add column property_is_private boolean;
+
+-- Update Asset IDs
+ALTER TABLE edc_asset_dataaddress DROP CONSTRAINT edc_asset_dataaddress_asset_id_fk_fkey;
+ALTER TABLE edc_asset_property DROP CONSTRAINT edc_asset_property_asset_id_fk_fkey;
+UPDATE edc_asset SET asset_id = REPLACE(REPLACE(asset_id, 'urn:artifact:', ''), ':', '-');
+UPDATE edc_asset_dataaddress SET asset_id_fk = REPLACE(REPLACE(asset_id_fk, 'urn:artifact:', ''), ':', '-');
+UPDATE edc_asset_property SET asset_id_fk = REPLACE(REPLACE(asset_id_fk, 'urn:artifact:', ''), ':', '-');
+ALTER TABLE edc_asset_dataaddress ADD CONSTRAINT edc_asset_dataaddress_asset_id_fk_fkey FOREIGN KEY (asset_id_fk) REFERENCES edc_asset (asset_id) ON DELETE CASCADE;
+ALTER TABLE edc_asset_property ADD CONSTRAINT edc_asset_property_asset_id_fk_fkey FOREIGN KEY (asset_id_fk) REFERENCES edc_asset (asset_id) ON DELETE CASCADE;
+
+UPDATE edc_asset_property SET property_name = 'https://w3id.org/edc/v0.0.1/ns/id', property_value=asset_id_fk WHERE property_name = 'asset:prop:id';
```

Comment:
starting 0.1.x all asset properties need to be URLs:

- MDS properties are already URLs.
- All asset:prop: fields need to be migrated.
- Fields like "description", "keyword", "langauge" might have EDC equivalents
- Custom fields might not, then we need to prepend a custom namespace.